backend/preprocessing/Makefile

```diff
@@ -1,2 +1,3 @@
 run:
 	poetry run python3 tst_preprocessing.py --save
+	poetry run python3 hysteresis_loops.py
```

Comment:
Prefer simply "python" instead of "python3" ?

Reply:
On Ubuntu Linux, `python` executable doesn't exist anymore. It's `python3`.
On the other hand, inside `poetry`, `python` exist.

I prefer to stay with `python3` that is more precise.